woocommerce/sensei-certificates
# Release 2.4.0

### Changes proposed in this Pull Request

* Bump the version
* Update the changelog
* Update the pot file
* Update the packages
* Some small maintenance changes

### Testing instructions

* Go to Plugins and make sure the version is `2.4.0`.
* Make sure the plugin works as before.
